src/cosima/inc/MCSource.hh

```diff
@@ -1,5 +1,5 @@
 /*
- * MCSource.hh
+ * MCParameterFile.hh
```

Comment:
Something strange happened through this class - looks like MCSource was replaced with MCParameterFile????